src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,152 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+import { getPeople } from '../../api';
+import { usePeoplePageContext }
+  from '../../pages/PeoplePage/PeoplePageContext/PeoplePageContext';
+import { Person } from '../Person';
+import { PeopleLoadingError } from '../PeopleLoadingError';
+import { NoPeopleOnServer } from '../NoPeopleOnServer';
+import { Loader } from '../Loader';
+import { SearchLink } from '../SearchLink';
+import { SortBy } from '../../types/FiltersType';
+
+export const PeopleTable: React.FC = () => {
+  const {
+    people,
+    loading,
+    setLoading,
+    setError,
+    setPeople,
+    error,
+  } = usePeoplePageContext();
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, [setError, setLoading, setPeople]);
+
+  const succesfulPeopleRender = !loading && !error && people.length > 0;
+  const appiSendNoPeople = !people.length && !loading;
+
+  const { sort, order } = usePeoplePageContext();
+
+  const getSortParams = (field: string) => {
+    if (field === sort && !order) {
+      return { sort: field, order: 'desc' };
+    }
+
+    if (field === sort && order) {
+      return { sort: null, order: null };
+    }
+
+    return { sort: field, order: null };
+  };
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {loading && <Loader />}
+        {error && <PeopleLoadingError />}
+        {appiSendNoPeople && <NoPeopleOnServer />}
+        {succesfulPeopleRender && (
+          <table
+            data-cy="peopleTable"
```

Comment:
Create a people table component, and this component looks like people page or something similar